components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -116,27 +180,31 @@ class SitePermissionsFeature(
     @Suppress("MaxLineLength")
     override fun onPermissionsResult(permissions: Array<String>, grantResults: IntArray) {
         sessionManager.runWithSessionIdOrSelected(sessionId) { session ->
```

Comment:
Now that we are use `BrowserStore`, I think we are not going to need `sessionManager`